lib/Conversion/PTensorToLinalg/PTensorToLinalg.cpp

```diff
@@ -300,7 +301,8 @@ static BodyType getBodyBuilder(::imex::ptensor::EWBinOpId bop,
   switch (bop) {
   case ptensor::ADD:
     return buildTrivial<mlir::arith::AddIOp, mlir::arith::AddFOp>(typ);
-  // case ptensor::ATAN2] =
+  case ptensor::ATAN2:
+    return buildTrivial<mlir::math::Atan2Op, void>(typ);
```

Comment:
This should probably be `buildTrivial<void, ::mlir::math::Atan2Op>(typ)`

Sooner rather than later we need to think about how to best do type conversions as required by the array-API. This probably needs to go somewhere before `buildTrivial`.